src/crypto/openssl/cose_sign.cpp

```diff
@@ -95,7 +136,7 @@ namespace ccf::crypto
     QCBOREncode_Init(&cbor_encode, signed_cose_buffer);
 
     t_cose_sign1_sign_ctx sign_ctx;
-    t_cose_sign1_sign_init(&sign_ctx, 0, T_COSE_ALGORITHM_ES256);
```

Comment:
Should we dynamically choose based on the key?..

Reply:
Yes